pomm-project/pomm-bundle/2.0/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "alias": "pomm",
```

Comment:
IMO the flex validation server should check this and forbid an alias on contrib repos, because aliases will be decided on the symfony/recipes only